src/app/modules/time-clock/components/entry-fields/entry-fields.component.ts

```diff
@@ -126,10 +130,36 @@ export class EntryFieldsComponent implements OnInit {
       this.entryForm.patchValue({ start_hour: this.newData.start_hour });
       return;
     }
+
+    this.getLastEntry();
+
+    const isFirstEntry = this.lastEntry !== undefined ? this.lastEntry.start_date : moment().add(-1, 'hours');
+    const isEntryDateInLastStartDate = moment(newHourEntered).isBefore(isFirstEntry);
+    if (isEntryDateInLastStartDate) {
+      this.toastrService.error('You cannot start a time-entry before another time-entry');
```

Comment:
What do you mean?

Reply:
Is to control that the user does not enter a time equal to or prior to the last time entry.
I have changed the message to: 
`There is already a time-entry with the time entered. The entry time cannot be the same as another time-entry`